client/trino-cli/src/main/java/io/trino/cli/QueryRunner.java

```diff
@@ -166,6 +175,32 @@ private static void setupBasicAuth(
         }
     }
 
+    private static void setupExternalAuth(
+            OkHttpClient.Builder builder,
+            ClientSession session,
+            boolean enabled,
+            Consumer<OkHttpClient.Builder> sslSetup)
+    {
+        if (enabled) {
+            checkArgument(session.getServer().getScheme().equalsIgnoreCase("https"),
+                    "Authentication using externalAuthentication requires HTTPS to be enabled");
+
+            RedirectHandler redirectHandler = uri -> {
```

Comment:
Duplicate of `RedirectMethod.CONSOLE` implementation?

Reply:
Good catch, RedirectMethod should be removed as the decision was to remove this option.